### DISCUSSION TITLE
AiI Stops after First Step? Anyone

### DISCUSSION THREAD

**Discussion by cocobeach** [stitionai/devika#349] [General]
I have never managed to go past the first planning steps, I ask it to continue but barely does what I ask for. It doesn't seem to use the APIs that are setup in the config.toml, part from the models and they all work, and I have setup almost everything, but it won't clone repositories and goes looking online for interacting with netlify.
The settings are still blank, has anyone solved any of those issues?
Grast OS tool by the way keep up the good job!

**Comment by obliviousz** [comment]
I also had the same issues. Did you install playwright? Does it do browser search etc?

**Reply by cocobeach** [reply]
Yes it browses writes a plan some code with placeholders, then keeps repeating that the task is done.
That got4, claude3 opus or sonnet and grow or local  mixtral .
